docs/source/cpp/parquet.rst

```diff
@@ -542,6 +542,19 @@ As an example, when serializing an Arrow LargeList to Parquet:
   :func:`ArrowWriterProperties::store_schema` was enabled when writing the file;
   otherwise, it is decoded as an Arrow List.
 
+Field Id
```

Comment:
This section should be moved below "Serialization details" below, so that the structure of the document makes sense (you can preview it in the GitHub UI).